### ISSUE TITLE
Move to ros_control project?

### ISSUE THREAD

**Issue by davetcoleman** [ros/filters#28]
I wasn't aware of this package until @gavanderhoorn pointed it out just now, but maybe this should live at https://github.com/ros-controls/ ? Thoughts @bmagyar?

**Comment by bmagyar** [comment]
First time to see this package for me too! 
I'd be up for hosting this in that org, no problem. It seems rather low-bandwidth at the moment, are you guys planning on extending it?

**Comment by tfoote** [comment]
It's currently in basic maintenance mode but would be happy to see it extended with new filters etc. There are lots of users outside of controls for example data processing. 

It might make more sense to create a downstream package `control_filters` that provides specific  filters for controls applications as plugins that can be used in the filter chains provided by this package.

**Comment by davetcoleman** [comment]
I think moving it to ``ros-control`` would provide a lot more visibility and usage, and prevent potential duplicate effort, but I have no immediate needs or use cases.

**Comment by peci1** [comment]
I use these filters to filter pointclouds/laserscans (via laser_filters package). It wouldn't make sense to me to depend on something from the ros-control repo.

**Comment by jonbinney** [comment]
@peci1 i think this is about moving to the ros-control org, not the ros_control repo. @davetcoleman is that right?

**Comment by peci1** [comment]
You're right, I got that wrong. But it still doesn't make sense to me as I treat filters as a generally usable package.

By the way, I recently released https://github.com/ctu-vras/sensor_filters which contains a set of nodes/nodelets ready-made for running filter chains on messages from sensor_msgs.

**Comment by bmagyar** [comment]
Would still be happy to host it at @ros-controls, we are planning some controllers for ros2 with these guys

**Comment by DLu** [comment]
I think it should stay here.
